### DISCUSSION TITLE
Update OneWire

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#542] [Ideas]
@MichaelDvP the headers for OneWire have been updated to support Arduino 2.0.3.

https://github.com/PaulStoffregen/OneWire/blob/v2.3.7/util/OneWire_direct_gpio.h

Do you think any of this is relevant to us? Since you merged this into OneWire,h before

**Comment by MichaelDvP** [comment]
Yes, the original lib does not work with new arduino core. I've taken the modified version from tasmota [https://github.com/arendst/Tasmota/tree/development/lib/lib_basic/OneWire-Stickbreaker](https://github.com/arendst/Tasmota/tree/development/lib/lib_basic/OneWire-Stickbreaker), with a few changes to make it compatible. But i think it's good to sync with the original from Paul Stoffregen again.
